### PR TITLE
doc: sep-2020 sec release pre-announcement

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by nschonni — APPROVED** [review]

**Comment by mhdawson** [comment]
Believe we have agreement these updates can be merged quickly based on previous review in nodejs-private. Will wait another 10 mins and then merge.

**Review by richardlau — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

File: build.js [hunk 1]
```diff
@@ -278,8 +278,8 @@ function getSource (callback) {
         },
         banner: {
           visible: false,
```
Comment (richardlau) [inline build.js:280]: Sorry, didn't spot this but I guess this should have been flipped to `true`?

Comment (richardlau) [reply build.js:280]: https://github.com/nodejs/nodejs.org/pull/3361

Comment (eoyx12) [reply build.js:280]: > Sorry, didn't spot this but I guess this should have been flipped to `true`?

**Pull request merged**